### PR TITLE
Updating registration policy text

### PR DESCRIPTION
**Pull request by hannestschofenig**

### CODE REVIEW

File: draft-ietf-scitt-architecture.md [hunk 1]
```diff
@@ -425,28 +426,34 @@ This attestation evidence SHOULD be supplemented with transparency Receipts for
...
-
-While this design ensures that all Verifiers get the same guarantee regardless of where a Transparent Statement is registered, its main downside is that it requires the Issuer to include the necessary policies in the Envelope when the Signed Statement is produced.
-Furthermore, it makes it impossible to register the same Signed Statement on two different Transparency Services, if their required Registration Policies are incompatible.
-
-{:aside}
-> **Editor's note**
->
-> The technical design for signalling and verifying registration policies is a work in progress.
+A Transparency Services that accepts to register any valid Signed
+Statement offered by anonymous Issuers would end up providing only
```
Comment (yogeshbdeshpande) [inline draft-ietf-scitt-architecture.md:430]:
```suggestion
Statement offered by anonymous Issuers would only provide 
```

File: draft-ietf-scitt-architecture.md [hunk 2]
```diff
@@ -425,28 +426,34 @@ This attestation evidence SHOULD be supplemented with transparency Receipts for
...
+limited value, or no value, to verifiers. As a consequence, some form of
+"verification" is needed before registration of Signed Statements to
```
Comment (yogeshbdeshpande) [inline draft-ietf-scitt-architecture.md:432]: Is it Verification or Authorization

**Review by yogeshbdeshpande — APPROVED** [review]
Left a minor comment, but in general LGTM!

File: draft-ietf-scitt-architecture.md (hunk 2 above)
Comment (hannestschofenig) [reply draft-ietf-scitt-architecture.md:432]: I would say "authorization" is better

**Review by OR13 — APPROVED** [review]

File: draft-ietf-scitt-architecture.md [hunk 3]
```diff
@@ -425,28 +426,34 @@ This attestation evidence SHOULD be supplemented with transparency Receipts for
...
-Furthermore, it makes it impossible to register the same Signed Statement on two different Transparency Services, if their required Registration Policies are incompatible.
-
-{:aside}
-> **Editor's note**
->
-> The technical design for signalling and verifying registration policies is a work in progress.
+A Transparency Services that accepts to register any valid Signed
+Statement offered by anonymous Issuers would only provide
+limited value, or no value, to verifiers. As a consequence, some form of
+"verification" is needed before registration of Signed Statements to
```
Comment (yogeshbdeshpande) [inline draft-ietf-scitt-architecture.md:432]:
```suggestion
"authorization" is needed before registration of Signed Statements to
```

**Review by henkbirkholz — APPROVED** [review]
Significant simplification, will require discussion time at IETF 117, probably.

**Pull request merged**